### PR TITLE
Update GraphQL query for address autocomplete

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Include option to request the type of address suggestions.

## Why?

In preparation for the next Embark package update.

_Referenced ticket(s): [GRW-366]_

[GRW-366]: https://hedvig.atlassian.net/browse/GRW-366

### CODE REVIEW

**Review by styrelius — APPROVED** [review]
🦅 💨

**Comment by robinandeer** [comment]
I guess I'll wait to merge this until we have the new Embark version included. Although nothing should break either way.

File: src/client/pages/Embark/addressAutocompleteProvider.ts [hunk 1]
```diff
@@ -17,13 +29,17 @@ const query = gql`
   }
 `
 
-export const resolveAddressAutocomplete = async (
+export const resolveAddressAutocomplete: AddressAutocompleteQuery = async (
   term: string,
-): Promise<AddressSuggestion[]> => {
+  { type } = { type: 'STREET' },
+) => {
```
Comment (gustaveen) [inline src/client/pages/Embark/addressAutocompleteProvider.ts:23]: Do we want to specify return type or is that unnecessary? :)

**Review by gustaveen — APPROVED** [review]
Great! Looks good to me! 🎉

File: src/client/pages/Embark/addressAutocompleteProvider.ts (hunk 1 above)
Comment (robinandeer) [reply src/client/pages/Embark/addressAutocompleteProvider.ts:23]: It's already defined in the type `AddressAutocompleteQuery`

**Comment by robinandeer** [comment]
Sorry for not waiting - all the buttons were green so I thought it meant it was reviewed 😅

**Pull request merged**